autoload/asyncomplete/utils.vim

```diff
@@ -0,0 +1,11 @@
+" Find a nearest to a `filename` directory `directoryname` by traversing the
+" filesystem upwards
+function! asyncomplete#utils#find_nearest_directory(filename, directoryname) abort
```

Comment:
can you change the function name to `find_nearest_parent_directory` to be more explicit.